### DISCUSSION TITLE
curerent version 1.1.2 dying with intellij version  2023.2 EAP (Ultimate Edition), had to be disabled

### DISCUSSION THREAD

**Discussion by greenhull** [chylex/IntelliJ-Inspection-Lens#16] [General]
IntelliJ IDEA 2023.2 EAP (Ultimate Edition)
Build #IU-232.6095.10, built on May 17, 2023
Licensed to IntelliJ IDEA EAP user: Chester John
Expiration date: June 16, 2023
Runtime version: 17.0.7+7-b953.1 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Windows 11.0
GC: G1 Young Generation, G1 Old Generation
Memory: 1262M
Cores: 16
Registry:
    debugger.new.tool.window.layout=true
    ide.experimental.ui=true
    ide.balloon.shadow.size=0

Non-Bundled Plugins:
    org.antlr.intellij.plugin (1.20)
    Gradle View (4.0.0)
    com.intellij.properties.bundle.editor (232.6095.10)
    org.sylfra.idea.plugins.linessorter (1.0.1)
    String Manipulation (9.7.1)
    com.github.platan.gradle-dependencies-formatter (0.6.1)
    tarn.gradle.dependencies (1.6.2)
    MavenRunHelper (4.23.222.2964.0)
    GrepConsole (12.18.211.6693.0)
    PlantUML integration (6.1.0-IJ2022.2)
    org.sonarlint.idea (8.2.0.68615)
    izhangzhihao.rainbow.brackets (2023.2.7)
    GradleDependenciesHelper (1.16)
    com.mallowigi (90.0.0)
    com.kn.diagrams.generator.generator (2022.2.0)

Kotlin: 232-1.8.21-release-380-IJ6095.10

**Comment by chylex** [comment]
Thanks for letting me know, moving the discussion to issue #17.
